var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -13,6 +13,7 @@ class PyNumpy(Package):
     extends('python')
     depends_on('py-nose')
     depends_on('netlib-blas+fpic', when='+blas')
+    depends_on('netlib-blas+fpic+fortran', when='+blas+fortran')
```

Comment:
Does this actually require specifically `netlib-blas`, or would any other BLAS work as well? All the Fortran LAPACK and BLAS routines have the same API, so using a non-Netlib BLAS is usually an order of magnitude faster.